src/connection.rs

```diff
@@ -262,6 +262,18 @@ impl Connection {
             packet.send(&mut self.tcp_stream)?;
 
             // TODO: S->C Player Abilities Packet
+
+            // Test Player Abilities Packet
+            // let mut packet = Packet::from_id_and_data( 
+            //     Varint(0x2E),
+            //     PacketData::Data(super::build_package_data!(
+            //         0b1101i8,
+            //         0.5f32,
+            //         0.5f32
+            //     )),
+            // );
+            // packet.send(&mut self.tcp_stream)?;
+
```

Comment:
Could you uncomment this and remove the `TODO: `?

Seems ready to me.

Reply:
Ok, but I'll better add another TODO at PacketData as a reminder to change the values.

Reply:
LGTM.